### ISSUE TITLE
Suggestion - Ingame WAD loading

### ISSUE THREAD

**Issue by bangstk** [fragglet/DietBoom#4]
I think that any discussion about "modernizing a sourceport to have good UI" ought to also concern how WADS are loaded, given how much of Doom playtime is spent in the mountain of user-made content.

In my opinion, the system of
- manually having to choose which IWAD is correct for a PWAD
- manually having to choose the correct complevel
- manually having to warp to the right map if the WAD doesnt start on MAP01

is probably one of the most archaic parts of playing Doom.

I think some simple work can be done to automate a lot of this process (e.g. see what maps are inside the loaded WAD, see which textures they use to decide if it needs to be launched with Plutonia or TNT instead of Doom 2, etc) rather than forcing the user to have to keep track of it.
In short, truly a 'load a WAD and play' experience.

I also think it would be even better if this was accessible from inside the game rather than all command line parameters. Several Quake sourceports are coming around to doing this.

And finally, I think some of the options like -fast, -respawn, -solonet, etc. ought also be accessible from ingame without needing the command line. And not buried within the options menu, but perhaps accessible right from the difficulty select screen.

Apologies that this is not an 'issue' per se, but I think it's useful discussion.
